Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5573,8 +5587,18 @@ RefPtr<StyleValue> Parser::parse_grid_track_placement_shorthand_value(Vector<Com
             tokens.skip_whitespace();
             current_token = tokens.next_token().token();
         }
-        if (current_token.is(Token::Type::Number) && current_token.number().is_integer())
+        if (current_token.is(Token::Type::Number) && current_token.number().is_integer()) {
+            // https://drafts.csswg.org/css-grid/#grid-placement-span-int
+            // Negative integers or zero are invalid.
+            if (has_span && static_cast<int>(current_token.number_value()) < 1)
+                return CSS::GridTrackPlacement(1, true);
             return CSS::GridTrackPlacement(static_cast<int>(current_token.number_value()), has_span);
+        }
+        // https://drafts.csswg.org/css-grid/#grid-placement-span-int
+        // If the <integer> is omitted, it defaults to 1.
+        if (has_span && current_token.to_string() == "/"sv) {
```

Comment:
I think is should be `current_token.token().delim() == '/'` rather than this (for the same `.to_string()` issues)

Reply:
(Probably `current_token.is(Token::Type::Delim) && current_token.token().delim() == '/'`)